developers/wallets/native/index.md

```diff
@@ -219,3 +219,129 @@ object containing the following details:
 ```
 
 See the [OID4VCI Specification](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html) for more details on this protocol.
+
+## Verifiable Credential Presentation
+
+Browser side verification flows will open the CHAPI wallet selector in the user's
+browser. If they select a native wallet, that URL will open the pre-registered
+native wallet apps link (see above for how to register...).
+
+That URL will to the Wallet's pre-registered domain--which will (thanks to the
+magic of app links!) open in the user's wallet:
+```
+https://wallet.example.com/switchboard?request=
+```
+
+Depending on the protocols supported, the value of request will contain one
+or more encoded JSON objects with the applicable details needed for using that
+protocol.
+
+{% mermaid %}
+sequenceDiagram
+  # User visits a Verifier's website and interacts with it to trigger a verification
+
+  participant site as Verifier Site
+  participant chapi as User's Browser / CHAPI
+  participant app as Native Wallet App
+  participant exchanger as VC-API Exchanger
+
+  site ->> exchanger: 1. creates an exchange within a workflow
+  exchanger ->> site: 2. JSON response contains the exchange URL/ID
+  site ->> chapi: 3. triggers CHAPI `get()` request w/blank VP + `protocols.vcapi` containing URL
```

Comment:
```suggestion
  site ->> chapi: 3. triggers CHAPI `get()` request w/blank `VerifiablePresentation` object + `protocols.vcapi` containing URL
```

Reply:
@brianorwhatever we're going to need code examples for this also, 'cause it's hard to even guess at what a "blank `VerifiablePresentation` object might be...but I'll commit the text change.